http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1242,6 +1236,12 @@ definitions:
         description: |
           Optional, the number of preemptible workers. If unspecified, the default number is 0. Ignored if numberOfWorkers is 0.
           For more information, see https://cloud.google.com/compute/docs/instances/preemptible
+      dataprocProperties:
```

Comment:
It looks like it expects `properties` in the json unless I'm mistaken?

Reply:
you're right...updated